modules/installation-guide/examples/che-proc_pulling-images-used-by-che-to-a-private-registry.adoc

```diff
@@ -52,11 +54,11 @@ $ docker run -it --rm \
 
 In the example above, substitute `<my-offline-registry>` for the image name and tag of the custom plug-in registry.
 
-=== Devfile base images
+.Devfile base images
 
 Every {prod-short} workspace uses one or more _base_ images, which contain the development dependencies for projects that are being built. To use the samples, pull these images from the offline devfile registry. This is needed because the sample devfiles included in the devfile registry apply to images suited for this purpose.
```

Comment:
> To use the samples, pull these images from the offline devfile registry.

More accurately, "To use the samples, Che will pull these images from the references to them in the devfile registry."

The only difference between the online and offline devfile registry, IIRC, is that one caches the sample projects and uses those zipped copies, whereas the online registry will cause the workspace to downloads from github on the fly when the workspace is instantiated.

Reply:
```suggestion
Every {prod-short} workspace uses one or more _base_ images, which contain the development dependencies for projects that are being built. To use the samples, {prod-short} pull these images from the references to them in the devfile registry. This is needed because the sample devfiles included in the devfile registry apply to images suited for this purpose.
```